docs/guides/recipe.rst

```diff
@@ -69,21 +74,52 @@ non-executable fields of ``package.json`` such as ``name``, ``version``,
 * **keywords**: A list of strings that are related to the package.
 * **maintainers**: A list of ``Maintainer`` objects that describe the contact
   information of the people responsible for the recipe (you, in this case).
-* **home**: ?
-* **sources**: ?
-* **icon**: A publicly-accessible URI to give the recipe a friendly face.
+* **home**: The homepage for this Recipe.
+* **sources**: A list of necessary container image URIs.
+* **icon**: A publicly-accessible image URI to give the recipe a friendly face.
 * **deprecated**: Whether or not the recipe is in active development.
 * **description**: A textual description, analogous to a README.
 * **license**: A valid `SPDX Identifier <https://spdx.org/licenses/>`_ which
```

Comment:
This has changed: https://pollination.github.io/queenbee/_static/redoc-recipe.html#tag/license_model